events/views.py

```diff
@@ -28,6 +28,11 @@ def time_direction(self):
         else:
             return settings.DEFAULT_TIME_DIRECTION
 
+    @property
+    def extra_context(self):
+        # TODO django 3.2 class based views update by this property, need to test on higher versions
```

Comment:
We should ticket this up if it's an action we need to take. The `todo` can be removed (they usually get forgotten about).

Having a quick look at the Django code base though - I can't see a reason why this wont work in higher version, `extra_context` is still available in Django 5.0

https://github.com/django/django/blob/main/django/views/generic/base.py#L29